datadump/global.c

```diff
@@ -56,6 +56,12 @@ const char * colors[] = {"TRANSPARENT", "DARK_GREY", "RED", "GREEN", "BLUE", "YE
 const char * buffers[] = {NULL, "PLAYER", "STR_VAR_1", "STR_VAR_2", "STR_VAR_3", NULL, "RIVAL", "VERSION",
                           "EVIL_TEAM", "GOOD_TEAM", "EVIL_LEADER", "GOOD_LEADER", "EVIL_LEGENDARY", "GOOD_LEGENDARY", NULL, NULL};
 
+const char * keygfx[] = {"A_BUTTON", "B_BUTTON", "L_BUTTON", "R_BUTTON", "START_BUTTON", "SELECT_BUTTON", "DPAD_UP", "DPAD_DOWN", "DPAD_LEFT", "DPAD_RIGHT", "DPAD_UPDOWN", "DPAD_LEFTRIGHT", "DPAD_ANY"};
+
+const char * extra[] = {"UP_ARROW_2", "DOWN_ARROW_2", "LEFT_ARROW_2", "RIGHT_ARROW_2", "PLUS", "LV_2", "PP", "ID", "NO", "UNDERSCORE", "CIRCLE_1", "CIRCLE_2", "CIRCLE_3", "CIRCLE_4", "CIRCLE_5", "CIRCLE_6", "CIRCLE_7", "CIRCLE_8", "CIRCLE_9", "LEFT_PAREN", "RIGHT_PAREN", "CIRCLE_DOT", "TRIANGLE", "BIG_MULT_X"};
+
+const char * emoji[] = {"EMOJI_UNDERSCORE", "EMOJI_PIPE", "EMOJI_HIGHBAR", "EMOJI_TILDE", "EMOJI_LEFT_PAREN", "EMOJI_RIGHT_PAREN", "EMOJI_UNION", "EMOJI_GREATER_THAN", "EMOJI_LEFT_EYE", "EMOJI_RIGHT_EYE", "EMOJI_AT", "EMOJI_SEMICOLON", "EMOJI_PLUS", "EMOJI_MINUS", "EMOJI_EQUALS", "EMOJI_SPIRAL", "EMOJI_TONGUE", "EMOJI_TRIANGLE_OUTLINE", "EMOJI_ACUTE", "EMOJI_GRAVE", "EMOJI_CIRCLE", "EMOJI_TRIANGLE", "EMOJI_SQUARE", "EMOJI_HEART", "EMOJI_MOON", "EMOJI_NOTE", "EMOJI_BALL", "EMOJI_BOLT", "EMOJI_LEAF", "EMOJI_FIRE", "EMOJI_WATER", "EMOJI_LEFT_FIST", "EMOJI_RIGHT_FIST", "EMOJI_BIGWHEEL", "EMOJI_SMALLWHEEL", "EMOJI_SPHERE", "EMOJI_IRRITATED", "EMOJI_MISCHIEVOUS", "EMOJI_HAPPY", "EMOJI_ANGRY", "EMOJI_SURPRISED", "EMOJI_BIGSMILE", "EMOJI_EVIL", "EMOJI_TIRED", "EMOJI_NEUTRAL", "EMOJI_SHOCKED", "EMOJI_BIGANGER"};
+
```

Comment:
Wrap these definitions' long lines like it is done for `buffers` above; the remainder of the codebase wraps at 160 columns